lib/package.gi

```diff
@@ -1099,6 +1099,19 @@ InstallGlobalFunction( IsPackageMarkedForLoading, function( name, version )
 ##
 #F  DefaultPackageBannerString( <inforec> )
 ##
+
+DeclareUserPreference( rec(
+  name:= "ShortBanners",
+  description:= [
+    "If this option is set to <K>true</K>, only the first line of each \
+package banner (including the name, version and description of the package) \
+is displayed."
+    ],
+  default:= false,
+  values:= [ true, false ],
+  multi:= false,
+  ) );
+
 InstallGlobalFunction( DefaultPackageBannerString, function( inforec )
```

Comment:
A couple packages "patch" the output of `DefaultPackageBannerString`. Changing its content globally would likely break those.

Instead of having `DefaultPackageBannerString` depend on a user preference, we could add a second optional argument `useShortBanner` (or whatever); if missing it'd default to `false`.

Then you can adjust the one call to `DefaultPackageBannerString ` in `lib/package.gi` to pass in `UserPreference( "ShortBanners" )` as that second argument.

Reply:
Done :-)